DummyData/dummyData.js

```diff
@@ -70,6 +105,11 @@ const songData = [
     fav_count: 69,
     path_to_artwork: 'https://i.imgur.com/AD3MbBi.jpeg',
     user_id: 'ClawzOut',
+    tags : {
+      id: "7b987c5b-6abe-4821-9315-99fc61781eb1",
+      song_id : "d76d275d-0c54-4f54-8627-edf27712195c",
+      name: "tag"
+    }
```

Comment:
I don't think the dummyData should have a different shape than the data in the database...so instead of including `tags` inside `songData`, tags should be held in their own dataset like they are in the database. Not a huge issue since I'm sure dummyData just gets used for testing or temporarily, but since we're here 🤷🏼‍♂️

Reply:
I think I see where this applies after looking at the way you query and join getSongsByUser. I suppose this works---though once the tag is attached to the song, we probably don't need the song_id...it's already in songData